experimental/adaptive-dialog/csharp_dotnetcore/calendar-bot-cui/Dialogs/CreateCalendarEntry/CreateCalendarEntry.cs

```diff
@@ -0,0 +1,315 @@
+                //    {
+                //        Steps = new List<IDialog>()
+                //        {
+                //            new SendActivity("[HelpCreateMeeting]")
+                //        }
+                //    },
+                //    new IntentRule("Cancel")
+                //    {
+                //        Steps = new List<IDialog>()
+                //        {
+                //                new SendActivity("[CancelCreateMeeting]"),
+                //                new EndDialog()
+                //        }
+                //    },
+                //    new IntentRule("ShowPrevious")
+                //    {
+                //        Steps = new List<IDialog>()
+                //        {
+                //        new IfCondition()
+                //        {
+                //            Condition = " 0 < user.CreateCalendarEntry_pageIndex",
+                //            Steps = new List<IDialog>()
+                //            {
+                //                new SetProperty()
+                //                {
+                //                    Property = "user.CreateCalendarEntry_pageIndex",
+                //                    Value = "user.CreateCalendarEntry_pageIndex - 1"
+                //                },
+                //                new RepeatDialog()
+                //            },
+                //            ElseSteps = new List<IDialog>()
+                //            {
+                //                new SendActivity("This is already the first page!"),
+                //                new RepeatDialog()
+                //            }
+                //        }
+                //        } 
+                //    },
+                //    new IntentRule("ShowNext")
+                //    {
+                //        Steps = new List<IDialog>(){
+                //            new IfCondition()
+                //            {
+                //                Condition = "user.CreateCalendarEntry_pageIndex * 3 + 3 < count(dialog.matchedEmails) ",
+                //                Steps = new List<IDialog>()
+                //                {
+                //                    new SetProperty()
+                //                    {
+                //                        Property = "user.CreateCalendarEntry_pageIndex",
+                //                        Value = "user.CreateCalendarEntry_pageIndex + 1"
+                //                    },
+                //                    new RepeatDialog()
+                //                },
+                //                ElseSteps = new List<IDialog>()
+                //                {
+                //                    new SendActivity("This is already the last page!"),
+                //                    new RepeatDialog()
+                //                }
+                //            }
+                //        }
+                //    }
+                //}
```

Comment:
Remove useless, the same.

Reply:
For the "help" and "cancel", if the rules are not enable, are these functional now?

---
In reply to: [308076854](https://github.com/microsoft/BotBuilder-Samples/pull/1612#discussion_r308076854) [](ancestors = 308076854)